pkg/combustion/templates/31-elemental-register.sh.tpl

```diff
@@ -47,5 +47,113 @@ Environment="CATTLE_AGENT_CONFIG=/etc/rancher/elemental/agent/config.yaml"
 ExecStart=/usr/sbin/elemental-system-agent sentinel
 EOF
 
+cat <<- EOF > /etc/systemd/system/elemental-reset.path
+[Path]
+PathModified=/var/lib/elemental/.unmanaged_reset
```

Comment:
I didn't tested it but I think it would be better to use "PathExists" so the service is only triggered once, otherwise as the service deletes it, it will be maybe trigger it again (?)
Quoting https://www.freedesktop.org/software/systemd/man/latest/systemd.path.html#PathExists=:

```
PathExists= may be used to watch the mere existence of a file or directory. If the file specified exists, the configured unit is activated.
...
PathChanged= may be used to watch a file or directory and activate the configured unit whenever it changes.
It is not activated on every write to the watched file but it is activated if the file which was open for writing gets closed.
```

Reply:
Thanks, Edu. I actually tested this quite extensively, and it behaves as expected here... when the file is created, it's triggered. Then the `elemental-reset.service` deletes the file as part of `ExecStartPost`.